Missing default `config.yml` in repo

**Describe the bug**
Missing default `config.yml` in repository causes `make docker-build-run` to fail:
```bash
Step 11/13 : COPY --from=build_base /tmp/go-gin-starter/config.yml /
COPY failed: stat tmp/go-gin-starter/config.yml: file does not exist
make: *** [docker-build-run] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:
1. `make docker-build-run`

**Expected behavior**
Successful build

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.7

--- a/_config.yml
+++ b/_config.yml
@@ -1,0 +1,9 @@
+server:
+  port: 8080
+
+database:
+  host: "localhost"
+  port: 5432
+  name: "dbname"
+  user: "dbuser"
+  password: "dbpassword"

